client/src/components/layouts/global.css

```diff
@@ -1,10 +1,16 @@
 .btn-cta-big {
   max-height: 100%;
-  height: 70px;
+  height: auto;
   font-size: 40px;
   white-space: normal;
 }
 
+@media (min-width: 768px) {
```

Comment:
768px  is correct as tablet width range starts from 768px. You can go through this article http://www.drupalthemer.org/blog/responsive-design-media-queries-and-how-to-make-them-more-friendly

Reply:
@rupali317 
Thanks for sharing. Good to be sure.